docs/designs/allreduce_based_horovod.md

```diff
@@ -0,0 +1,117 @@
+# Design for Elastic AllReduce-based Training Support Based on Horovod
+
+This document describes the design for supporting AllReduce-based distributed
+training based on Horovod in ElasticDL.
```

Comment:
for suppporting AllReduce distribution strategy.

and please give a link to Horovod.